primitives/runtime/src/traits.rs

```diff
@@ -94,6 +94,46 @@ impl IdentifyAccount for sp_core::ecdsa::Public {
 	}
 }
 
+/// Means of signature verification.
+///
+/// Accepts custom hashing fn for the message and custom convertor fn for the signer.
+pub trait CustomVerify<
```

Comment:
IMO the naming is weird and as this seems to be only BEEFY specific, can we not just put this into some BEEFY crate?

Reply:
Ok. Yes, maybe it's better to put it in a beefy crate or just define a verify method in `beefy_primitives::crypto`. I'll adjust the PR.

Reply:
Done. I moved the trait to `primitives\beefy` and adjusted and renamed it.